.github/workflows/draft-release.yml

```diff
@@ -85,14 +85,19 @@ jobs:
     needs: draft-release
     strategy:
       matrix:
-        distro:
-        - debian-stretch
-        - debian-buster
-        - debian-sid
-        - ubuntu-xenial
-        - ubuntu-bionic
-        - ubuntu-focal
+        dbld-builder-images:
+        - [ 'debian-stretch',   'deb' ]
+        - [ 'debian-buster',    'deb' ]
+        - [ 'debian-sid',       'deb' ]
+        - [ 'ubuntu-xenial',    'deb' ]
+        - [ 'ubuntu-bionic',    'deb' ]
+        - [ 'ubuntu-focal',     'deb' ]
+        - [ 'centos-7',         'rpm' ]
+        - [ 'fedora-33',        'rpm' ]
       fail-fast: false
+    env:
+      DISTRIBUTION: ${{ matrix.dbld-builder-images[0] }}
+      PACKAGE_MANAGER: ${{ matrix.dbld-builder-images[1] }}
```

Comment:
If a package is a `deb` is should not mean that the upload is needed. As I only see use case for uploading or not, this should represent just that and not package manager.

This could be resolved either having yet an other job that only uploads (and the distro list would contains the distroes that needs uploading) or you could just make the 2nd parameter like upload: yes/no.

Reply:
I've added an "upload" flag for each matrix element.
Laci will refactor this part even further when he returns from vacation.

https://github.com/MrAnno/syslog-ng/actions/runs/1210275712